google/cloud/bigtable/examples/bigtable_hello_world.cc

```diff
@@ -80,10 +80,10 @@ int main(int argc, char* argv[]) try {
         google::cloud::bigtable::SetCell("family", "c0", greeting)));
     ++i;
   }
-  //! [write rows]
+  //! [write rows] [END writing_rows]
 
   // Read a single row.
-  //! [read row]
+  //! [read row] [START getting_a_row]
   auto result = table.ReadRow(
       "key-0",
       google::cloud::bigtable::Filter::ColumnRangeClosed("family", "c0", "c0"));
```

Comment:
The examples in other language separate the filter creation to a separate region tag.  Would it be worth it to move this line to its own line so we can tag it?